### PR TITLE
Update to rococo-v1

### PR DESCRIPTION
**Pull request by riusricardo**

### CODE REVIEW

File: 1-prep/1-compiling.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ git clone  https://github.com/paritytech/cumulus
 cd cumulus
 
 # Checkout the proper commit
-git checkout 4786231
+git checkout xxxxxxx
```
Comment (apopiak) [inline 1-prep/1-compiling.md:82]: uhm, that seems like it needs to be filled in?

File: 3-parachains/1-launch.md [hunk 2]
```diff
@@ -33,14 +33,16 @@ We can now start the collator node with the following command. Notice that we're
...
 ```bash
-parachain-collator \
+rococo-collator \
+  --collator
   --tmp \
-  --ws-port 9977 \
-  --port 30336 \
-  --parachain-id 200 \
-  --validator \
+  --parachain-id \
```
Comment (apopiak) [inline 3-parachains/1-launch.md:39]: parachain id is missing here

Comment (apopiak) [inline 3-parachains/1-launch.md:39]:
```suggestion
  --parachain-id 200 \
```

File: 1-prep/2-chain-spec.md [hunk 3]
```diff
@@ -85,34 +85,34 @@ spec file ccan be reproduced. The second part is obtained similarly with `//Bob`
...
   Account ID:       0xbe5ddb1579b72e84524fc29e78609e3caf42e85aa118ebfe0b0ad404b5bdd25f
-  SS58 Address:     15Jbynf3EcRqdHV1K14LXYh7PQFTbp5wiXfrc4kbMReR9KxA
+  SS58 Address:     5GNJqTPyNqANBkUVMN1LPPrxXnFouWXoe2wNSmmEoLctxiZY
 ```
 
 Polkadot grandpa session key for `//Alice` (`ed25519` cryptography).
 
 ```bash
-$ subkey inspect --scheme ed25519 --network polkadot //Alice
+$ subkey inspect --scheme ed25519 --network substrate //Alice
```
Comment (JoshOrndorff) [inline 1-prep/2-chain-spec.md:99]:
```suggestion
$ polkadot key inspect --scheme ed25519 --network substrate //Alice
```
These are relay chain values. And `polkadot` now includes the `key` subcommand. If you use it instead, you can save the user from having to build `subkey`

**Pull request merged**